hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataWriteUtils.java

```diff
@@ -200,6 +200,11 @@ public static HoodieWriteConfig createMetadataWriteConfig(
           builder.withProperties(datadogConfig.build().getProps());
           break;
         case PROMETHEUS:
+          HoodieMetricsPrometheusConfig prometheusConfig = HoodieMetricsPrometheusConfig.newBuilder()
+              .withPushgatewayLabels(writeConfig.getPushGatewayLabels())
+              .withPrometheusPortNum(writeConfig.getPrometheusPort()).build();
```

Comment:
why we are not setting other props like host, jobname, etc.

Reply:
I checked Prometheus reporter and we need only prometheus port and push gateway labels